src/quo2/components/lowest_price.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.lowest-price
+  (:require [quo.design-system.colors :as colors]
+            [quo.react-native :as rn]))
+
+(defn line-styles [flex-grow]
+  {:opacity 0.4
+   :border-style :dotted
+   :border-width 1
+   :border-color (:text-01 @colors/theme)
+   :flex-grow flex-grow
+   :overflow :hidden
+   :border-radius 1})
+
+(def centrify {:flex-direction :row
+               :justify-content :center
+               :align-items :center})
+
+(defn border-alignment [color]
+  {:align-self :flex-start
+   :padding-horizontal 2
+   :padding-vertical 2
+   :flex-grow 0.25
+   :border-radius 3
+   :background-color color})
+
+(defn lowest-price
+  [{:keys [top-value
+           bottom-value
+           margin-top
+           top-value-text-color
+           top-value-bg-color
+           bottom-value-bg-color
```

Comment:
We should only provide inputs for dynamic values like label. Props like, colors should be calculated automatically based on theme.

Reply:
Okay so they aren't supposed to be configurable .. I see